src/providers/SignatureProvider.tsx

```diff
@@ -60,4 +139,4 @@ const useSignatureContext = () => {
     return useContext(SignatureContext);
 };
 
-export { SignatureContext, SignatureProvider, useSignatureContext };
+export { SignatureContext, SignatureProvider, useSignatureContext, type InputElementOptions };
```

Comment:
not sure if the reserved word type is needed here but that's just my observation, might be wrong 😶‍🌫️

Reply:
It is a reserved word! But we're using it to declare that `InputElementOptions` is a type that we're exporting 

Syntax highlighting might help visualize it! 
<img width="751" alt="image" src="https://user-images.githubusercontent.com/52969178/235568815-aaedf3ab-2df1-4a85-848e-3044030492d9.png">

Reply:
Ok so for exporting an interface, the type reserved word is used, gotcha 😉